modules/lymphgen/1.0/lymphgen.smk

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env snakemake
+import os
+import pandas
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Chris Rushton
+# Module Author:    Chris Rushton
+# Contributors:     NA
+
+# LymphGen classifier writen by George Wright
+# DO NOT DISTRIBUTE THE LYMPHGEN SOURCE CODE WITHOUT GEORGE'S PERMISSION
+# This module has been brought to you by Krysta's Girl Guide cookies. Helping me
+# miss all weight loss goals since 2018
+
+##### SETUP #####
+
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CFG`
+# `CFG` is a shortcut to `config["lcr-modules"]["lymphgen"]`
+CFG = op.setup_module(
```

Comment:
How does this behave without a samples table? Or do you always use one to prevent this from erroring out?